src/BUILD

```diff
@@ -1281,6 +1334,100 @@ cc_test(
     copts = COPTS_TESTS,
 )
 
+cc_library(
+    name = "test_utils",
+    linkstatic = 1,
+    hdrs = [
+        "test/c_api_test_utils.hpp",
+        "test/test_utils.hpp",
+        "test/stress_test_utils.hpp",
+        ],
+    srcs = [
+        "test/test_utils.cpp",
+        ],
+    data = [
+    ],
+    linkopts = [
+        "-lxml2",
+        "-luuid",
+        "-lstdc++fs",
+        "-lcrypto",
+        "-lssl",
+    ],
+    deps = [
+        "//src:ovms_lib",
+        "@com_google_googletest//:gtest",
+    ],
+    copts = COPTS_TESTS,
+)
+
+cc_library(
+    name = "libtest_environment",
```

Comment:
It tests gpu environment ? change file and project name then to gpu/va/ocl ? Environment to common name.